cmd/validate-krew-manifest/main.go

```diff
@@ -77,6 +78,16 @@ func validateManifestFile(path string) error {
 	}
 	glog.Infof("structural validation OK")
 
+	if maxLen := 80; !validateLineLength(p.Spec.Description, maxLen) {
+		return fmt.Errorf("line length in `description` exceeds %d characters", maxLen)
```

Comment:
why don't we use `pkg/errors`?

Reply:
I think as long as the error message is specific enough, it is more user-friendly to have a single line instead of a lengthy stack trace.

Reply:
SGTM